### DISCUSSION TITLE
How to quick start with debug this project.

### DISCUSSION THREAD

**Discussion by liudonghua123** [nocodb/nocodb#8340] [Q&A] [answered]
Hi, I just want to contribute some features/code. I find if I modified some code, `localhost:8080` doesn't reflect the changes. I have to change `start:frontend` to `"start:frontend": "pnpm --filter=nc-gui run build:copy",` and make the `nc-lib-gui` dependence of nocodb to `"nc-lib-gui": "workspace:^",`. However I have to generate the static files, it's too wasting time.

If the backend can proxy the static files requests to frontend served on `localhost:3000`, then the dev experience will be efficient and quickly.

**Comment by mertmit** [comment, marked as answer]
Hi @liudonghua123,
On development setup you should access through `localhost:3000`, this way you will see your changes.
Please revert package.json changes, run both of the commands and access through `localhost:3000`.
